app/views/helpers/export/articles.phtml

```diff
@@ -18,51 +18,20 @@ if (empty($this->entryIdsTagNames)) {
 	$this->entryIdsTagNames = array();
 }
 
-foreach ($this->entriesRaw as $entryRaw) {
-	if ($entryRaw == null) {
+foreach ($this->entries as $entry) {
+	if ($entry == null) {
 		continue;
 	}
-	$entry = FreshRSS_Entry::fromArray($entryRaw);
-	if (!isset($this->feed)) {
-		$feed = FreshRSS_CategoryDAO::findFeed($this->categories, $entry->feed());
-		if ($feed === null) {
-			$feed = $entry->feed(true);
-		}
-	} else {
-		$feed = $this->feed;
-	}
 
-	$article = array(
-		'id' => $entry->guid(),
-		'timestampUsec' => '' . $entry->id(),
-		'categories' => array_values($entry->tags()),
-		'title' => $entry->title(),
-		'author' => $entry->authors(true),
-		'published' => $entry->date(true),
-		'updated' => $entry->date(true),
-		'alternate' => array(array(
-			'href' => htmlspecialchars_decode($entry->link(), ENT_QUOTES),
-			'type' => 'text/html',
-		)),
-		'content' => array(
-			'content' => $entry->content(),
-		),
-		'origin' => array(
-			'streamId' => $feed == null ? '' : $feed->id(),
-			'title' => $feed == null ? '' : $feed->name(),
-			'htmlUrl' => $feed == null ? '' : $feed->website(),
-			'feedUrl' => $feed == null ? '' : $feed->url(),
-		)
-	);
-	$article['categories'][] = $entry->isRead() ? 'user/-/state/com.google/read' : 'user/-/state/com.google/unread';
-	if ($entry->isFavorite()) {
-		$article['categories'][] = 'user/-/state/com.google/starred';
-	}
-	$tagNames = isset($this->entryIdsTagNames['e_' . $entry->id()]) ? $this->entryIdsTagNames['e_' . $entry->id()] : array();
-	foreach ($tagNames as $tagName) {
-		$article['categories'][] = 'user/-/label/' . $tagName;
+	$feed = $this->feed ?? FreshRSS_CategoryDAO::findFeed($this->categories, $entry->feedId());
+	$entry->_feed($feed);
+
+	if (isset($this->entryIdsTagNames['e_' . $entry->id()])) {
+		$entry->_tags($this->entryIdsTagNames['e_' . $entry->id()]);
```

Comment:
This wrongly mixed tags and labels. Regression fixed in https://github.com/FreshRSS/FreshRSS/pull/5626